spec/Candid-Motoko.md

```diff
@@ -1,32 +1,32 @@
-The IDL-Motoko integration
```

Comment:
Sorry, I hadn't seen this before. I actually think that this file should stay in the Motoko repo. It's not part of the Candid spec.

Reply:
It's arguable, and can stay in either repo. I think it has value to stay here, as it provides a good example of how Candid integrates with other languages, and how to deal with type gaps. So I slightly prefer it to stay here. WDYT?

Reply:
Maybe we should have one for Rust and JS (where the implemenation is in this repo), as good examples :-)

Reply:
The Rust one is harder to write, as it needs to fight between nominal typing and structural typing. Motoko-Candid is actually the simplistic example.

Reply:
@rossberg Do you want to remove this from the repo? I'm fine either way, but need a decision to merge this PR.

Reply:
[Sorry, day off last Friday.] Yes, I think this should be removed here, but that can be a separate PR.